### PR TITLE
Add neutron_qe script tests

### PR DESCRIPTION
**Pull request by fyanac**

### CODE REVIEW

File: tests/roles/dataplane_adoption/tasks/neutron_verify.yaml [hunk 1]
```diff
@@ -52,3 +52,10 @@
   when: prelaunch_test_instance|bool
   ansible.builtin.shell: |
       ping -c4 {{ lookup('env', 'FIP') | default('192.168.122.20', True) }}
+
+- name: stop ping
+  when: neutron_qe_test|bool
+  environment:
+    OS_CLOUD_IP: "{{ standalone_ip | default(edpm_node_ip) }}"
+  shell: |
```
Comment (holser) [inline tests/roles/dataplane_adoption/tasks/neutron_verify.yaml:60]: ansible.builtin.shell

File: tests/roles/development_environment/tasks/main.yaml [hunk 2]
```diff
@@ -61,3 +61,25 @@
       alias openstack="ssh -i {{ edpm_privatekey_path }} -o StrictHostKeyChecking=no ${SOURCE_OSP_SSH_USER}@${OS_CLOUD_IP} OS_CLOUD=${OS_CLOUD_NAME} openstack"
       ${BASH_ALIASES[openstack]} credential create admin test -f value -c id
   register: before_adoption_credential
+
+- name: Execute create resources script
+  when: neutron_qe_test|bool
+  environment:
+    OS_CLOUD_IP: "{{ standalone_ip | default(edpm_node_ip) }}"
+  shell: |
```
Comment (holser) [inline tests/roles/development_environment/tasks/main.yaml:69]: ansible.builtin.shell

File: tests/roles/development_environment/tasks/main.yaml [hunk 3]
```diff
@@ -61,3 +61,25 @@
...
+      ssh ${OS_CLOUD_IP} "set -o pipefail && chmod 744 {{ neutron_qe_dir }}/create_resources.sh && \
+      {{ neutron_qe_dir }}/create_resources.sh > {{ neutron_qe_dir }}/create_resources.sh.log 2>&1"
+- name: Execute prepare-pinger script
+  when: neutron_qe_test|bool
+  environment:
+    OS_CLOUD_IP: "{{ standalone_ip | default(edpm_node_ip) }}"
+  shell: |
```
Comment (holser) [inline tests/roles/development_environment/tasks/main.yaml:76]: ansible.builtin.shell

File: tests/roles/development_environment/tasks/main.yaml [hunk 4]
```diff
@@ -61,3 +61,25 @@
...
+      ssh ${OS_CLOUD_IP} "set -o pipefail && chmod 744 {{ neutron_qe_dir }}/prepare-pinger.sh && \
+      {{ neutron_qe_dir }}/prepare-pinger.sh > {{ neutron_qe_dir }}/prepare-pinger.sh.log 2>&1"
+- name: Execute start-pinger script
+  when: neutron_qe_test|bool
+  environment:
+    OS_CLOUD_IP: "{{ standalone_ip | default(edpm_node_ip) }}"
+  shell: |
```
Comment (holser) [inline tests/roles/development_environment/tasks/main.yaml:83]: ansible.builtin.shell

File: tests/roles/dataplane_adoption/tasks/neutron_verify.yaml (hunk 1 above)
Comment (fyanac) [reply tests/roles/dataplane_adoption/tasks/neutron_verify.yaml]: Thank you

File: tests/roles/dataplane_adoption/tasks/neutron_verify.yaml [hunk 5]
```diff
@@ -52,3 +52,82 @@
...
+
+    - name: get validation pinger result
+      ansible.builtin.shell: |
+          ssh ${OS_CLOUD_IP} "set -o pipefail && grep -r 'res=1' {{ neutron_qe_dir }}/validate_pinger.sh.log"
+      register: validation_pinger_result
+      ignore_errors: true
+
+    - name: verify pinger result
+      ansible.builtin.assert:
+      that:
```
Comment (cescgina) [inline tests/roles/dataplane_adoption/tasks/neutron_verify.yaml:78]: this should be indented one more level

File: tests/roles/dataplane_adoption/tasks/neutron_verify.yaml [hunk 6]
```diff
@@ -52,3 +52,82 @@
...
+
+    - name: get validation result_reboot
+      ansible.builtin.shell: |
+          set -o pipefail && grep -r 'fail' ~/validations_after/connectivity_summary_file_reboot
+      register: validation_result_reboot
+      ignore_errors: true
+
+    - name: verify result_reboot
+      ansible.builtin.assert:
+      that:
```
Comment (cescgina) [inline tests/roles/dataplane_adoption/tasks/neutron_verify.yaml:104]: same as above, indentation is off

File: tests/roles/dataplane_adoption/tasks/neutron_verify.yaml [hunk 7]
```diff
@@ -52,3 +52,82 @@
...
+
+    - name: get validation result_migration
+      ansible.builtin.shell: |
+          set -o pipefail && grep -r 'fail' ~/validations_after/connectivity_summary_file_migration
+      register: validation_result_migration
+      ignore_errors: true
+
+    - name: verify result migration
+      ansible.builtin.assert:
+      that:
```
Comment (cescgina) [inline tests/roles/dataplane_adoption/tasks/neutron_verify.yaml:124]: ditto

File: tests/roles/dataplane_adoption/tasks/neutron_verify.yaml (hunk 6 above)
Comment (fyanac) [reply tests/roles/dataplane_adoption/tasks/neutron_verify.yaml]: Thank you

**Comment by cescgina** [comment]
recheck

**Comment by cescgina** [comment]
recheck

**Review by marios — APPROVED** [review]
lgtm and jobs green (neutron_qe test is false there)

**Review by cescgina — APPROVED** [review]
lgtm

**Review by jistr — APPROVED** [review]

**Pull request merged**